docs/api/ComponentEventsObserver.md

```diff
@@ -10,7 +10,7 @@
 
 ## bindComponent
 
-`bindComponent(component: Component<any>): EventSubscription`
+`bindComponent(component: Component<any>, componentIdArg: string): EventSubscription`
```

Comment:
Mark also as optional here?

Reply:
@henrikra What's the preferred way to mark that argument as optional?

Reply:
`componentIdArg?: string` like this :)